spec/features/chefs/ingredients/index_spec.rb

```diff
@@ -0,0 +1,21 @@
+require 'rails_helper'
+
+RSpec.describe 'Ingredients index page', type: :feature do
+  before do
+    @chef = Chef.create!(name: 'Gordon Ramsay')
+    @dish = @chef.dishes.create(name: 'Beef Stew')
+    @ingredient1 = Ingredient.create(name: 'Beef')
+    @ingredient2 = Ingredient.create(name: 'Potatoes')
+    @ingredient3 = Ingredient.create(name: 'Carrots')
+    @dish.ingredients << [@ingredient1, @ingredient2, @ingredient3]
+  end
+
+  it 'displays the chef name and ingredients list' do
+    visit "/chefs/#{@chef.id}/ingredients" 
+    save_and_open_page
+    expect(page).to have_content("Ingredients Used by #{@chef.name}")
+    expect(page).to have_content(@ingredient1.name)
+    expect(page).to have_content(@ingredient2.name)
+    expect(page).to have_content(@ingredient3.name)
+  end
```

Comment:
In order to take this test further, I would create some additional ingredients and not associate them to the chef's dish and assert that they don't show up on the page. This would make it completely obvious that it's just not all the ingredients in the database showing up on the page but only the ones associated with the chef's dishes.